integration_tests/nlg_meta_eval_test.py

```diff
@@ -39,3 +40,94 @@ def test_da_cs_en(self):
 
         self.assertGreater(len(sys_info.results.analyses), 0)
         self.assertGreater(len(sys_info.results.overall), 0)
+
+
+class MetaEvalNLGTest(unittest.TestCase):
```

Comment:
Add tests to check that specifying invalid values of either `level` or `func_name` causes an error.